rubygem-smart_proxy_dynflow_core/rubygem-smart_proxy_dynflow_core.spec

```diff
@@ -109,6 +111,7 @@ install -Dp -m0644 %{buildroot}%{gem_instdir}/deploy/%{service_name}.service %{b
 %{root_sysconfdir}/%{gem_name}/settings.yml
 %doc %{gem_instdir}/LICENSE
 %{root_bindir}/%{service_name}
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
```

Comment:
I don't think I agree, this file should be updated if possible, I don't think it makes sense for a user to manage it.